editor/frontend/src/TransformElementsTool.ts

```diff
@@ -310,6 +333,125 @@ export class MoveElementsTool extends TransformElementsTool {
   }
 }
 
+/** @alpha Create new elements with translation applied to placement. */
+export class CopyElementsTool extends MoveElementsTool {
+  public static override toolId = "CopyElements";
+  public static override iconSpec = "icon-move"; // Need better icon...
+
+  public static override get minArgs() { return 0; }
+  public static override get maxArgs() { return 1; }
+
+  private _numCopiesProperty: DialogProperty<number> | undefined;
+  public get numCopiesProperty() {
+    if (!this._numCopiesProperty)
+      this._numCopiesProperty = new DialogProperty<number>(
+        PropertyDescriptionHelper.buildNumberEditorDescription("numCopies", EditTools.translate("CopyElements.Label.NumCopies"),
+          { type: PropertyEditorParamTypes.Range, minimum: 1 } as RangeEditorParams), 1);
+    return this._numCopiesProperty;
+  }
+
+  public get numCopies(): number { return this.numCopiesProperty.value; }
+  public set numCopies(value: number) { this.numCopiesProperty.value = value; }
+
+  protected override get wantMakeCopy(): boolean { return this.numCopies > 0; }
+
+  protected override updateAnchorLocation(transform: Transform): void {
+    // Account for additional copies for repeat operation anchor point...
+    for (let iCopy = 0; iCopy < this.numCopies; ++iCopy)
+      super.updateAnchorLocation(transform);
+  }
+
+  protected async doTranformedCopy(ids: Id64Array, transform: Transform, numCopies: number): Promise<Id64Arg | undefined> {
+    // NOTE: Using loadProps to return json format geometry to the frontend for each element in the tool agenda is very inefficient.
```

Comment:
Can you move this important disclaimer into the documentation for the class?

Reply:
done.